config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,13 @@
+CarrierWave.configure do |config|
+  if Rails.env.test?
+    config.storage = :file
+  else
+    config.fog_credentials = {
+      provider:              'AWS',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.